### DISCUSSION TITLE
Can not change UNIQUE KEY to Merge-On-Write mode

### DISCUSSION THREAD

**Discussion by LCehoennardo** [apache/doris#45624] [A - General / Q&A]
将一个unique模型的表从Merge-On-Read转为Merge-On-Write报错：
```sql
alter table dor_unique_table_rt set ("enable_unique_key_merge_on_write"="true")
```

报错：
```
Execution failed: Error Failed to execute sql: java.sql.SQLException: (conn=848962) errCode = 2, detailMessage = Can not change UNIQUE KEY to Merge-On-Write mode
```

这个参数不支持更改吗？

**Comment by tyzh3323** [comment]
不支持mor和mow的互换吧

**Reply by LCehoennardo** [reply]
哪里有说明吗

**Reply by ixzc** [reply]
you can refer this：Use attention
https://doris.apache.org/zh-CN/docs/table-design/data-model/unique

**Reply by LCehoennardo** [reply]
看到了 thank you
<img width="1017" alt="image" src="https://github.com/user-attachments/assets/311efc1f-7023-4ff8-8cdd-dac52a9f9f8f" />

**Comment by echo-hhj** [comment]
不支持mor表转换成mow表，只能重新建表。
